supervision/detection/tools/inference_slicer.py

```diff
@@ -9,6 +9,7 @@
 from supervision.detection.utils import move_boxes, move_masks, move_obb_boxes
 from supervision.utils.image import crop_image
 from supervision.utils.internal import SupervisionWarnings
+from supervision import config
```

Comment:
Let's only import `ORIENTED_BOX_COORDINATES`, rather than the whole config. This matches the pattern in the rest of the repo.